src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -120,29 +124,36 @@ function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
         // the first line is not the header row so we need to strip it
         rawCSV = rawCSV.substring(rawCSV.indexOf("\n") + 1);
 
-        const parsedArray = csvParse(rawCSV, (row) => {
-          const transformedRow = autoType(row);
-          // autotype won't catch formatted thousands and percentages,
-          // so run them through numeral
-          Object.entries(transformedRow).forEach(([key, value]) => {
-            if (value && typeof value === "string") {
-              const transformedValue = numeral(value).value();
-              transformedRow[key] =
-                transformedValue === null ? value : transformedValue;
-            }
-          });
-
-          return transformedRow;
-        })
+        const parsedArray = csvParse(
+          rawCSV,
+          (row): CountyLevelRecord => {
+            // we only need a few columns, which we will explicitly format
+            const totalIncarceratedPopulation = numeral(
+              row["Total Incarcerated Population"],
```

Comment:
Is there any chance this is ever null or empty? Will `numeral` auto convert this to 0?

Reply:
no, numeral will output `null` if the value is missing. looks like I'm not handling this consistently throughout this function — the `@types` definition for `numeral().value()` appears to actually be wrong, which is part of why I didn't catch this. I'll take another pass at these and make sure there are sensible fallback values for every `numeral`